src/vs/platform/theme/common/colorRegistry.ts

```diff
@@ -388,6 +387,12 @@ export const treeIndentGuidesStroke = registerColor('tree.indentGuidesStroke', {
 export const tableColumnsBorder = registerColor('tree.tableColumnsBorder', { dark: '#CCCCCC20', light: '#61616120', hc: null }, nls.localize('treeIndentGuidesStroke', "Tree stroke color for the indentation guides."));
 export const listDeemphasizedForeground = registerColor('list.deemphasizedForeground', { dark: '#8C8C8C', light: '#8E8E90', hc: '#A7A8A9' }, nls.localize('listDeemphasizedForeground', "List/Tree foreground color for items that are deemphasized. "));
 
+/**
+ * Quick pick widget (dependent on List and tree colors)
+ */
+export const _deprecatedQuickInputListFocusBackground = registerColor('quickInput.list.focusBackground', { dark: null, light: null, hc: null }, nls.localize('quickInput.listFocusBackground', "Quick picker background color for the focused item."), undefined, nls.localize('depre', "Please use quickInputList.focusBackground instead"));
```

Comment:
- I would leave the description empty 
- Improve the nls key for the deprecated label (right now it's `depre`)